spec/signable/query/client_spec.rb

```diff
@@ -125,17 +126,39 @@ def create_envelope_params(template_fingerprint: nil, party_id: nil) # rubocop:d
   describe '#create' do
     context 'when sending request to create an envelope' do
       context 'when existant template and party are provided' do
-        it 'returns message confirming the envelope has been created', vcr: 'client/envelopes/create/success' do
-          response = client.create('envelopes', create_envelope_params)
+        subject(:create_envelope) do
+          client.create('envelopes', create_envelope_params(auto_expire_hours: auto_expire_hours))
+        end
 
-          expect(response).to be_instance_of(Signable::Query::Response)
-          expect(response.ok?).to eq(true)
-          expect(response.http_response['envelope_title']).to eq('whatever')
-          expect(response.http_response['message'])
-            .to eq('Your envelope with title whatever will be processed and sent out.')
+        context 'without auto expire hours' do
+          let(:auto_expire_hours) { nil }
 
-          client.cancel('envelopes', response.http_response['envelope_fingerprint'])
-          client.delete('envelopes', response.http_response['envelope_fingerprint'])
+          it 'returns message confirming the envelope has been created', vcr: 'client/envelopes/create/success' do
+            expect(create_envelope).to be_instance_of(Signable::Query::Response)
+            expect(create_envelope.ok?).to eq(true)
+            expect(create_envelope.http_response['envelope_title']).to eq('whatever')
+            expect(create_envelope.http_response['message'])
+              .to eq('Your envelope with title whatever will be processed and sent out.')
+
+            client.cancel('envelopes', create_envelope.http_response['envelope_fingerprint'])
+            client.delete('envelopes', create_envelope.http_response['envelope_fingerprint'])
+          end
+        end
+
+        context 'with auto expire hours' do
+          let(:auto_expire_hours) { 120 }
+
+          it 'returns message confirming the envelope has been created',
+             vcr: 'client/envelopes/create/auto_expire_hours_success' do
+            expect(create_envelope).to be_instance_of(Signable::Query::Response)
+            expect(create_envelope.ok?).to eq(true)
+            expect(create_envelope.http_response['envelope_title']).to eq('whatever')
+            expect(create_envelope.http_response['message'])
+              .to eq('Your envelope with title whatever will be processed and sent out.')
```

Comment:
Is there nothing in the response that indicates the auto_expire_hours has been accepted?

Reply:
Unfortunately not =/. If you check the `auto_expire_hours_success.yml` file, you'll be able to see the request and the response for the create request. `Signable` send us the same response as the traditional `create`. So, to make sure that the API request was working correctly, I had to check the `Signable` Dashboard directly.